src/System.Xml.XDocument/tests/Properties/DeepEquals.cs

```diff
@@ -1,407 +1,390 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.IO;
 using System.Linq;
 using System.Collections;
 using System.Collections.Generic;
-using System.Xml;
-using System.Xml.Linq;
-using Microsoft.Test.ModuleCore;
+using CoreXml.Test.XLinq;
+using Xunit;
 
-namespace CoreXml.Test.XLinq
+namespace System.Xml.Linq.Tests
 {
-    public partial class FunctionalTests : TestModule
+    public class DeepEqualsTests
     {
-        public partial class PropertiesTests : XLinqTestCase
+        // equals => hashcode should be the same
+
+        //  - all "simple" node types
+        //  - text vs. CDATA
+        //  XDocument:
+        //  - Normal mode
+        //  - Concatenated text (Whitespace) nodes
+        //  - Diffs in XDecl
+
+        //  XElement:
+        //  - Normal mode
+        //      - same nodes, different order
+        //          - comments inside the texts
+        //      - same nodes, same order (positive)
+        //
+        //  - Concatenated text nodes
+        //      - string content vs. text node/s
+        //          - empty string vs. empty text node
+        //      - Multiple text nodes but the same value
+        //      - adjacent text & CData
+        //  
+        //  - IsEmpty
+        //  - Attribute order
+        //  - Namespace declarations 
+        //      - local vs. in-scope
+        //      - default redef.
+
+        [InlineData("PI", "click", "PI", "click", true)]        // normal
+        [InlineData("PI", "PI", "PI", "PI", true)]              // target=data
+        [InlineData("PI", "", "PI", "", true)]                  // data = ''
+        [InlineData("PI", "click", "PI", "", false)]            // data1!=data2
+        [InlineData("AAAAP", "click", "AAAAQ", "click", false)] // target1!=target2
+        [InlineData("AAAAP", "AAAAQ", "AAAAP", "AAAAQ", true)]  // hashconflict I
+        [InlineData("PA", "PA", "PI", "PI", false)]             // data=target
+        [InlineData("AAAAP", "AAAAQ", "AAAAQ", "AAAAP", false)] // hashconflict II
+        [Theory]
+        public void ProcessingInstruction(string target1, string data1, string target2, string data2, bool checkHashCode)
         {
-            public partial class DeepEqualsTests : XLinqTestCase
```

Comment:
>There are a few large blocks of code being deleted here

I recall only one [such place](https://github.com/dotnet/corefx/pull/3854/files?w=1#diff-013916273010d7a23c6415c3d2c05e77L47). These code not used anywhere.


>ImplicitConversionsElem

All code there has relevant replacements except a couple test cases
https://github.com/dotnet/corefx/pull/3854#discussion_r42038208
https://github.com/dotnet/corefx/pull/3854#discussion_r42038377

>DeepEqualsTests 

All code there has relevant replacements